### PR TITLE
(Website) Fix broken links

### PR DESCRIPTION
**Pull request by Aegide**

### CODE REVIEW

**Comment by williewillus** [comment]
can you link to `master` instead? It'll redirect to the default branch so we don't have to keep updating it

File: web/faq.html [hunk 1]
```diff
@@ -38,7 +38,7 @@
 The Mana Pool holds a <i>lot</i> of mana. Flowers like the Hydroangeas produce very small amounts of it, so you probably won't see a difference. Go get some better flowers (like Endoflames).</li><br>
 
 <li id="q10-fake"><b>I found a bug, how do I report it?</b> <a href="#q10">(permalink)</a><br>
-Report it on the <a href="https://github.com/Vazkii/Botania/issues">issue tracker</a>. Please read the <a href="https://github.com/Vazkii/Botania/blob/master/.github/CONTRIBUTING.md">contributor guidelines</a> first, though!</li><br>
+Report it on the <a href="https://github.com/Vazkii/Botania/issues">issue tracker</a>. Please read the <a href="https://github.com/VazkiiMods/Botania/blob/1.18.x/CONTRIBUTING.md">contributor guidelines</a> first, though!</li><br>
```
Comment (williewillus) [inline web/faq.html]: VazkiiMods on the first one, and use master branch on the second one

File: web/faq.html [hunk 2]
```diff
@@ -47,7 +47,7 @@
 <a href="{% link terrablade.html %}">...</a></li><br>
 
 <li id="q15-fake"><b>Is there a Botania API? Can I make a Botania Addon?</b> <a href="#q15">(permalink)</a><br>
-Yes, there is <a href="https://github.com/Vazkii/Botania/tree/master/src/main/java/vazkii/botania/api">an API</a>. You <i>can</i> make addons, but please give me a heads-up about what you're going to add. I can't disallow you from adding things, given the mod's license, but I'd at least like to know what where you're going with it.</li><br>
+Yes, there is <a href="https://github.com/VazkiiMods/Botania/tree/1.18.x/Xplat/src/main/java/vazkii/botania/api">an API</a>. You <i>can</i> make addons, but please give me a heads-up about what you're going to add. I can't disallow you from adding things, given the mod's license, but I'd at least like to know what where you're going with it.</li><br>
```
Comment (williewillus) [inline web/faq.html]: use master

**Review by williewillus — CHANGES_REQUESTED** [review]

File: web/faq.html (hunk 1 above)
Comment (williewillus) [reply web/faq.html]: You missed this one

Comment (Aegide) [reply web/faq.html]: sorry about that, fixed

File: web/faq.html (hunk 2 above)
Comment (Aegide) [reply web/faq.html]: sure

**Review by Aegide — COMMENTED** [review]
Ok should be valid now.

**Pull request merged**